DSharpPlus/Entities/Interaction/DiscordInteractionDataOption.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities
+{
+    /// <summary>
+    /// Represents parameters for interaction commands. 
+    /// </summary>
+    public sealed class DiscordInteractionDataOption
+    {
+        /// <summary>
+        /// Gets the name of this interaction parameter.
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; internal set; }
+
+        /// <summary>
+        /// Gets the option type of this interaction parameter. 
+        /// <para>This can be cast to an <see langword="int"></see>, <see langword="bool"></see>, <see langword="string"></see> or <see langword="ulong"/> depending on the <see cref="DiscordInteractionDataOption.Name"/></para>
+        /// </summary>
+        [JsonProperty("value")]
+        [JsonConverter(typeof(DiscordInteractionOptionTypeConverter))]
+        public object OptionType { get; internal set; }
```

Comment:
Shouldn't this be named `Value` and not `OptionType`?

Reply:
Yes that would probably be a good idea.